GuiBibeau/akeru
# Hero Section Button Bg Issue Solved #68

## Description of the pull request

This pull request addresses the issue with the background gradient class used in the HTML. The problem arose when the bottom background color of the gradient matched the button background color, causing visual blending, especially on devices with low screen height. To mitigate this issue, the background color of the HTML tag has been slightly lightened to create a clearer distinction between the background and button colors on such devices.

Additionally, the footer background color has been removed to ensure a seamless transition between the modified background color of the HTML tag and the footer, preventing any visual inconsistencies caused by conflicting background colors.

## Changes made

- Adjusted the background color of the HTML tag to provide better contrast with the button background color on low screen height devices.
- Removed the footer background color to maintain visual harmony with the modified background color of the HTML tag.

## Related issues

Closes #68

## Testings done

To test this PR:
1. View the website on devices with low screen height to verify improved contrast between background and button colors.
2. Check for any visual inconsistencies between the modified HTML background color and the footer.

## Screenshots (if any)

![image](https://github.com/GuiBibeau/akeru/assets/110533554/48c8effc-8a04-4172-9423-cbf784d0573f)
![image](https://github.com/GuiBibeau/akeru/assets/110533554/ad888a6e-ab9b-4879-bff1-f36e7b9a7cfe)


## Checklist

- [ ] My code does not produce new errors
- [ ] I gave myself a code review before asking others.
